src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static final Map<String, Integer> STORAGE = new HashMap<>();

```

Comment:
this is not really a constant since its content is changing all the time
```suggestion
    public static final Map<String, Integer> storage = new HashMap<>();
```

Reply:
fixed)